index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet"> 
+    <title>My Portoflio</title>
+</head>
+<body>
+    <header>
+        <h4 id="welcome-text">Welcome</h4>
+        <nav>
+            <img src="img/Union.png" alt="">
```

Comment:
- [ ] [OPTIONAL] We can also have this hamburger image inside a clickable element since it is something we will need to click on in order to show the hidden menu links, you can have something like this 👉  `<button><img src="img/Union.png" alt=""></button>`